lib/scope.js

```diff
@@ -1681,75 +1681,52 @@ module.exports = {
   },  // Ends scope.today_to_date()
 
   findFiles: async function( scope, {to_find_names}) {
+    /** Returns a list of paths to the sources folder associated
+    *    with the given list of file names.
+    * 
+    * @param to_find_names { [str] } List of file names
+    * 
+    * @returns { [str] } sources folder paths
+    */
     // First make sure the list of files was defined. Don't remove spaces yet -
     // allow a typo where someone forgot to include a space after a comma.
+    // TODO: wish we had the var name here that folks are trying to set
     if ( to_find_names.length === 0 ) {
       reports.addToReport( scope, {
         type: `error`,
-        value: `Your list of files to find is empty.`
+        value: `Your list of files is empty.`
```

Comment:
Not sure I get this change. Why would you remove this extra information? Without context, the former report msg is clearer IMO.

Reply:
To get into this nit, if you're interested, this is just my gut feeling: I feel it muddies the waters because it doesn't necessarily line up with the step they wrote. Getting inaccurately specific can sometimes be more confusing than being more general. They either gave file(s) to "upload" or they gave file(s) to "compare". They didn't mention "finding" anything and they don't know what the code is doing in the background. Again, just my gut feeling. We could try asking a user, though, either now or at some future point.

Reply:
I get that, but I don't think this make it any clearer to the user. If I was a user, the first thing I'd think if I saw this error message, I'd think "what list?". If they only intended to pass one file to upload (even if they didn't pass any)? In their mind, they didn't pass a list, so this message is still confusing. It should be followed by a specific error message from the upload or compare steps anyway.

I guess this gets into who the messages are for. When they contain relatively little information like this, IMO they should at least be useful to developers. Several months out, the former message would be still be useful to me, whereas the latter wouldn't, and depending on how the user is communicating their issue with them, it either adds a simple search, or a lot more back and forth than necessary.

Reply:
Good point. What do you think of `There are no file(s) to find written in your Step.`? I'll push that for now, dependent on your input.

Reply:
`Your Step is missing the names of which file(s) to find.`

Reply:
I like the `Your step is missing...` one!